indico/migrations/versions/20220830_1428_0c44046dc1be_add_generate_accompanying_person_tickets.py

```diff
@@ -0,0 +1,26 @@
+"""Add generate accompanying person tickets
+
+Revision ID: 0c44046dc1be
+Revises: b45847c0e62f
+Create Date: 2022-08-30 14:28:32.880804
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '0c44046dc1be'
+down_revision = 'b45847c0e62f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('forms', sa.Column('tickets_for_accompanying_persons', sa.Boolean(), nullable=False,
+                                     server_default='true'), schema='event_registration')
```

Comment:
I would default to `false` here since that way there's no change in behavior for existing events.